### PR TITLE
[Calls] Continue improving documentation and general information

### PR DESCRIPTION
**Pull request by renandincer**

### CODE REVIEW

**Review by astroza — APPROVED** [review]

File: content/calls/introduction.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- **It's difficult to scale WebRTC**: Many struggle scaling WebRTC servers. Operators run into issues about how many users can be in the same "room" or want to build unique solutions that don't fit into the current concepts in high level APIs.
+
+- **High egress costs**: WebRTC is expensive to use as managed solutions charge a high premium on cloud egress and running your own servers incur system administration and scaling overhead. Cloudflare already has 300+ locations with upwards of 1,000 servers in some locations. Cloudflare Calls scales easily on top of this architecture and can offer the lowest WebRTC usage costs.
+
+- **WebRTC is growing**: Developers are realizing that WebRTC is not just for video conferencing. WebRTC is supported on many platforms, it mature and well understood.
+
+## What makes Cloudflare Calls unique
+
+- **Unopinonated*: Cloudflare Calls does not offer a SDK. It instead allows you to access raw WebRTC to solve unique problems that might not fit into existing concepts. API is deliberately simple.
```
Comment (nils-ohlmeier) [inline content/calls/introduction.md]: "API is ..." -> "It's API is ..." or "The API is ..."

File: content/calls/introduction.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+- **SFU**: Calls is a special kind of pub/sub server that is good at forwarding media data to clients that subscribe to certain data. Each client connects to Cloudflare Calls via WebRTC and either sends data, receives data or both usign WebRTC. This can be audio/video tracks or DataChannels.
+
+- **It scales**: All Cloudflare servers act as a single server so millions of WebRTC clients can connect to Cloudflare Calls. Each can send data, receive data or both with other clients.
+
+## How most developers get started
+
+1. Get started with the echo example, which you can download from the Cloudflare dashboard when you create a Calls App or from [demos](/calls/demos/). This will show you how to send and receive audio and video.
+
+2. Understand how you can manuplate who can recieve what media by passing around session and track ids. Remember, you control who receives what media. Each media track is represented by a unique ID. It's your responsibility to save and distribute this ID.
```
Comment (nils-ohlmeier) [inline content/calls/introduction.md]: s/manuplate/manipulate/

File: content/calls/limits.md [hunk 3]
```diff
@@ -1,20 +1,25 @@
 ---
 pcx_content_type: get-started
-title: Limits and timeouts
+title: Limits, timeouts, quotas
```
Comment (nils-ohlmeier) [inline content/calls/limits.md]: I think this read better as "Limits, timeouts and quotas" as you spelled it out below.

**Review by nils-ohlmeier — APPROVED** [review]
Looks good, just a few minor suggestions.

**Pull request merged**